src/SortableList/index.js

```diff
@@ -38,73 +39,62 @@ class SortableList extends PureComponent {
 
   componentDidUpdate(prevProps) {
     if (prevProps.list.rows !== this.props.list.rows && !!this._list) {
+      this.cache.clearAll();
       this._list.wrappedInstance.recomputeRowHeights();
     }
   }
 
-  renderRow({ index, key, style }) {
+  renderRow({ index, key, style, parent}) {
     const row = this.props.list.rows[index];
-
-    return (
-      <SortableItem
-        key={row.id}
-        row={row}
-        rowId={row.id}
-        listId={this.props.listId}
-        rowStyle={style}
-        itemComponent={this.props.itemComponent}
-        moveRow={this.props.moveRow}
-        dropRow={this.props.dropRow}
-        dragBeginRow={this.props.dragBeginRow}
-        dragEndRow={this.props.dragEndRow}
-        findItemIndex={this.props.findItemIndex}
-        dndDisabled={this.props.dndDisabled}
-      />
-    );
-  }
-
-  renderItemForMeasure({ rowIndex }) {
-    const { itemComponent: DecoratedItem } = this.props;
-    const row = this.props.list.rows[rowIndex];
-
     return (
```

Comment:
what did decoratedItem not do that cell measurer does?

Reply:
This `renderItemForMeasure` was getting used by the old CellMeasurer API as a kind of dummy component for obtaining row heights. It would pass a callback into the `itemComponent` called `getRowHeight` which would allow the item to get it's height directly from the CellMeasurer